src/Package/MSBuild.UnGAC.Setup/MSBuild.UnGAC.Setup.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <!-- The project MUST be net45. -->
+    <TargetFramework>net45</TargetFramework>
+    <!-- Set as an exe because this project publishes its own output. -->
+    <OutputType>Exe</OutputType>
+    <!-- Forcing the 'vsix' output to be json output. -->
```

Comment:
Add

> ... workaround for dotnet/arcade#xxxx

(and file the "support exe packages" bug there ;))

Reply:
Filed: https://github.com/dotnet/arcade/issues/6120